<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KCL Visible Collision Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);</string>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #2c3e50; background-color: #edf0f1; border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>KCL Visible Collision Tool</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="lightToggle">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>170</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox { color: #2c3e50; }
QCheckBox::indicator { width: 18px; height: 18px; }
QCheckBox::indicator:unchecked { border: 2px solid #3498db; background-color: white; }
QCheckBox::indicator:checked { border: 2px solid #3498db; background-color: #3498db; }</string>
    </property>
    <property name="text">
     <string>Add lightning effect</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>700</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px dashed #3498db; border-radius: 10px; background-color: #e8f6ff; color: #2c3e50;</string>
    </property>
    <property name="text">
     <string>Click or Drag &amp; Drop SZS file here...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>700</width>
      <height>40</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
                QProgressBar {
                    border: 2px solid #3498db;
                    border-radius: 5px;
                    text-align: center;
                    background-color: #e8f6ff;
                }
                QProgressBar::chunk {
                    background-color: #2ecc71;
                    border-radius: 3px;
                }
            </string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>700</width>
      <height>100</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
                color: #2c3e50;
                font-size: 14px;
                font-weight: bold;
            </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="downloadBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>700</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
                QFrame {
                    border: 2px solid #2ecc71;
                    border-radius: 10px;
                    background-color: #e8f6ff;
                }
            </string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="downloadLabel">
       <property name="styleSheet">
        <string notr="true">
                            border: none;
                            font-size: 18px;
                            font-weight: bold;
                            color: #2c3e50;
                        </string>
       </property>
       <property name="text">
        <string>Your file is ready!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="styleSheet">
        <string notr="true">
                            QPushButton {
                                background-color: #2ecc71;
                                color: white;
                                border-radius: 5px;
                                padding: 10px;
                                font-size: 14px;
                                font-weight: bold;
                                max-width: 200px;
                            }
                            QPushButton:hover {
                                background-color: #27ae60;
                            }
                            QPushButton:pressed {
                                background-color: #229954;
                            }
                        </string>
       </property>
       <property name="text">
        <string>Download Result</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="mainButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>470</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #2ecc71; color: white; border-radius: 5px; }
QPushButton:hover { background-color: #27ae60; }
QPushButton:pressed { background-color: #229954; }</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>700</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="pathForm">
     <item row="0" column="0">
      <widget class="QLabel" name="blender_label">
       <property name="text">
        <string>Blender's (3.x) path:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="blenderLayout">
       <item>
        <widget class="QLineEdit" name="blenderPathEdit">
         <property name="styleSheet">
          <string notr="true">padding: 5px; border: 1px solid #bdc3c7; border-radius: 3px;</string>
         </property>
         <property name="placeholderText">
          <string>Enter Blender path...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="blenderCheckButton">
         <property name="text">
          <string>Check/Install</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="brawlcrate_label">
       <property name="text">
        <string>BrawlCrate's path:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="brawlCrateLayout">
       <item>
        <widget class="QLineEdit" name="brawlCratePathEdit">
         <property name="styleSheet">
          <string notr="true">padding: 5px; border: 1px solid #bdc3c7; border-radius: 3px;</string>
         </property>
         <property name="placeholderText">
          <string>Enter BrawlCrate path...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="brawlCrateCheckButton">
         <property name="text">
          <string>Check/Install</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>800</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="link_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;a href=&quot;https://www.dropbox.com/scl/fo/ed3jhx5plohx5smtua6f1/h?rlkey=bokbpopgb3un0pqy7ss7k4jxs&amp;e=2&amp;st=oafu6csk&amp;dl=0&quot;&gt;All 32 tracks with visible collision by Epik&lt;/a&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextBrowserInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSZS"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGithub"/>
    <addaction name="actionDiscord"/>
    <addaction name="actionTutorial"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSZS">
   <property name="text">
    <string>Load SZS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionDiscord">
   <property name="text">
    <string>Shadow's Discord</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="text">
    <string>Manual Tutorial (by Epik95)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
